code-of-conduct.md

```diff
@@ -66,42 +56,39 @@ that are not aligned to this Code of Conduct, or to ban temporarily or
 permanently any community member for other behaviors that they deem
 inappropriate, threatening, offensive, or harmful.
 
-## Enforcement
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported in the following ways. You may send an open message from any channel
-to all Community Helpers by tagging @community helpers. You may send an open
-message from any channel or a direct message to @kattni#1507, @tannewt#4653,
-@Dan Halbert#1614, @cater#2442, sommersoft#0222, or @Andon#8175. You may
-also send an email to <support@adafruit.com>. All complaints will
-be reviewed and investigated, and will result in a response that is deemed
-necessary and appropriate to the circumstances. The moderation team is
-obligated to maintain confidentiality with regard to the reporter of an
-incident. Be aware, if you send a message in an open channel, your report will
-not be anonymous. In situations where the issue is particularly egregious,
-requires immediate action, and no moderators are available, you can also report
-the message directly to Discord.
-
-Further details of specific enforcement policies are outlined in the
-Moderation section.
-
 ## Moderation
 
-These are the policies for upholding our community’s standards of conduct.
+Instances of behaviors that violate the Adafruit CircuitPython Code of Conduct
+may be reported by any member of the community. Community members are
+encouraged to report these situations, including situations they witness
+involving other community members.
+
+You may report in the following ways:
+
+In any situation, you may send an email to <support@adafruit.com>.
 
-1. Messages, comments, nicknames or avatar images that violate the Adafruit
-CircuitPython standards of conduct, including anything hateful, hurtful,
-oppressive, or exclusionary, are not allowed.
-2. Messages, comments, nicknames or avatar images that moderators find
-inappropriate, whether listed in the code of conduct or not, are also not
-allowed.
+On the Adafruit Discord, you may send an open message from any channel
+to all Community Helpers by tagging @community helpers. You may also send an
+open message from any channel or a direct message to @kattni#1507, 
+@tannewt#4653, @Dan Halbert#1614, @cater#2442, @sommersoft#0222, or @Andon#8175.
+
+In situations on Discord where the issue is particularly egregious, requires
+immediate action, and no moderators are available, you can also report the
```

Comment:
I'd say "should" instead of "can". Maybe "particularly egregious, possibly illegal, requires immediate action or violates the Discord terms of service, you should also report the message directly to Discord."

Reply:
Agreed. I'll reword this.